browser/ui/views/location_bar/brave_search_conversion/promotion_button_controller.cc

```diff
@@ -134,16 +137,20 @@ bool PromotionButtonController::ShouldShowSearchPromotionButton() {
   // Promotion button will be shown for current search provider's
   // suggestion entries to make users search with brave search with that
   // suggestion.
-  if (!omnibox_view_->model()->PopupIsOpen()) {
+  OmniboxEditModel* edit_model = browser_->window()
+                                     ->GetLocationBar()
+                                     ->GetOmniboxController()
+                                     ->edit_model();
+  if (!edit_model->PopupIsOpen()) {
```

Comment:
It seems `PopupIsOpen()` is moved to `OmniboxController::IsPopupOpen()`.

https://chromium-review.googlesource.com/c/chromium/src/+/7135420

Reply:
Added another cr144 commit: https://github.com/brave/brave-core/pull/32452/commits/f55527c3ecc82bbada5bef87b9da4a70f6c8111a